.github/workflows/deploy-preview.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy Preview
+
+on:
+  pull_request:
+
+jobs:
+  deploy-preview:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
```

Comment:
The same comment as CI PR, do we need it here? And should we change to use `setup-node` action?